src/shared/util.js

```diff
@@ -51,6 +51,17 @@ const AnnotationMode = {
   ENABLE_STORAGE: 3,
 };
 
+const AnnotationEditorPrefix = "pdfjs_internal_editor_";
+
+const AnnotationEditorType = {
+  NONE: 0,
+  FREE_TEXT: 1,
+};
+
+function isValidAnnotationEditorMode(mode) {
+  return mode >= 0 && mode < Object.keys(AnnotationEditorType).length;
```

Comment:
It seems generally safer, with any future changes, to use  `Object.values(AnnotationEditorType).includes(mode)` here similar to the pre-existing helper functions.